backend/bitboxbase/bitboxbase.go

```diff
@@ -38,25 +42,57 @@ type Interface interface {
 
 	// BlockInfo returns some blockchain information.
 	BlockInfo() string
+
+	// GetIP implement a getter for the IP under which the base is reachable
+	GetIP() string
+
+	// GetElectrsRPCPort implements a getter for the electrs rpc port
+	GetElectrsRPCPort() string
+
+	// GetNetwork implements a getter for the network type, either mainnet or testnet
+	GetNetwork() string
 }
 
 //BitBoxBase provides the dictated bitboxbase api to communicate with the base
 type BitBoxBase struct {
 	bitboxBaseID    string //This is just the ip at the moment, but will be an actual unique string, once the noise pairing is implemented
 	registerTime    time.Time
+	ip              string
 	closed          bool
 	updaterInstance *updater.Updater
+	electrsRPCPort  string
+	network         string
+	log             *logrus.Entry
 }
 
 //NewBitBoxBase creates a new bitboxBase instance
 func NewBitBoxBase(ip string, id string) (*BitBoxBase, error) {
 	bitboxBase := &BitBoxBase{
+		log:             logging.Get().WithGroup("bitboxbase"),
 		bitboxBaseID:    id,
 		closed:          false,
-		updaterInstance: updater.NewUpdater(),
+		ip:              strings.Split(ip, ":")[0],
```

Comment:
it is confusing that `ip` can mean with or without port depending on the context. Better stick to one way only.


elsewhere we use `address string` to be the endpoint including port.

speaking of: why

Reply:
Currently we still communicate with servers running on other ports on the BitBox Base. This will change once everything is hosted through a single port.